osu.Game.Rulesets.Rush/Beatmaps/RushCraftedBeatmapConverter.cs

```diff
@@ -67,7 +67,6 @@ namespace osu.Game.Rulesets.Rush.Beatmaps
     /// 1) Slider velocity and BPM are currently ignored, but may be used in the future
     /// 2) Kiai will be used in the future to indicate boss sections
     /// 3) New combo is ignored
-    /// 4) Hitsounds are only transferred for minions
```

Comment:
Oh, thought it wasn't intentional but this comment says it, wonder why?

Reply:
Originally hitsounds were used to select object types, so they couldn’t be passed through after conversion. The logic has changed to be mostly based on position, so it’s fine to transfer hitsounds now.

Reply:
I see, makes sense.